src/server/cmd/pachtf/main.go

```diff
@@ -51,10 +52,10 @@ type Entrypoint = func(ctx context.Context, log *logrus.Logger, args []string) e
 var entrypoints = map[string]Entrypoint{
 	"sql-ingest":      sqlIngest,
 	"sql-gen-queries": sqlGenQueries,
+	"sql-run":         sqlRun,
```

Comment:
should maybe add some comments indicating that the first two are only preserved for backwards compatibility